batch.md

```diff
@@ -48,52 +62,76 @@ DataSource<EventType> dataSet = env.createInput(inputFormat, TypeInformation.of(
                                    .setParallelism(2);
 ...
 ```
+### Deserialization
 
-### Parameters

```

Comment:
I understand you have renamed Parameters section to Builder API. I feel the name `Parameters` seems more general than calling it Builder API. We anyway make a reference to the builder API in the explanation part. Also, I think it will be easy for someone to follow the Parameters immediately after the code since it maintains the flow. The same comment is applicable for the `streaming.md` file as well.

Reply:
Accepted suggestion